packages/cli/package.json

```diff
@@ -119,6 +119,7 @@
     "solc-wrapper": "^0.5.8",
     "solidity-parser-antlr": "^0.4.2",
     "spinnies": "^0.3.0",
+    "topological-sort": "^0.3.0",
```

Comment:
I'd rather we used [`toposort`](https://www.npmjs.com/package/toposort), with millions of weekly downloads, than this [relatively unused library](www.npmjs.com/package/topological-sort).

Reply:
I won't block on this though, and I'm not sure how strict you often are about this sort of thing. @ylv-io?

Reply:
Let's use `topsort`. It is a right thing to do.